awx/ui/client/lib/components/action/action-button.partial.html

```diff
@@ -0,0 +1,3 @@
+<button class="btn at-Button{{ vm.fill }}--{{ vm.color }}">
```

Comment:
change to {{ vm.color ? `--${vm.color}` : '' }} and this will pass the working at-Button class when neither fill or color are passed

Reply:
👍I will do that.